.github/workflows/tag.yml

```diff
@@ -83,13 +83,6 @@ jobs:
             rsync -d -r dist \
             gs://$BUCKET_OPENFIN/tag/${{ steps.vars.outputs.TAG }}
 
-      - name: Deploy (Launcher)
```

Comment:
about time we retired this .. very old installations used to point to `launcher.prod.reactivetrader.com` ..